content/en/agent/guide/how_remote_config_works.md

```diff
@@ -147,11 +152,13 @@ After you perform these steps, your Agent requests its configuration from Datado
 [1]: /getting_started/site/
 [2]: /help/
 [3]: /account_management/api-app-keys/
```

Comment:
NIT: I wonder if it should be 
```suggestion
[3]: /account_management/rbac/permissions#api-and-application-keys
```
as a link here, as we reference the api_key permission.